components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -347,6 +391,17 @@ func (p *pg) associatePolicyWithProjects(ctx context.Context,
 	return nil
 }
 
+func (p *pg) notifyPolicyChange(ctx context.Context, q Querier) error {
+	// We keep track of an id with each change. This lets us be smart about only updating
+	// the OPA rules when it might change.
+	_, err := q.ExecContext(ctx, "UPDATE policy_change_tracker SET policy_change_id = uuid_generate_v4();")
```

Comment:
I think you might be able to say `UPDATE policy_change_tracker SET policy_change_id = DEFAULT;` but we would lose explicitness there so prob not worth it